packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -44,7 +44,16 @@
               v-if="$slots['toolbar-button']"
               class="toolbar-button-container"
             >
-              <slot name="toolbar-button" />
+              <Teleport
+                v-if="!useActionOutside"
+                to="#action-button"
```

Comment:
will need to change the selector here as well

Reply:
Updated